src/XrdAccSciTokens.cc

```diff
@@ -684,4 +756,31 @@ XrdAccAuthorize *XrdAccAuthorizeObject(XrdSysLogger *lp,
     }
 }
 
+XrdAccAuthorize *XrdAccAuthorizeObjectAdd(XrdSysLogger *lp,
+                                          const char   *cfn,
+                                          const char   *parm,
+                                       XrdAccAuthorize *accP)
+{
+    // Record the parent authorization plugin. There is no need to use
+    // unique_ptr as all of this happens once in the main and only thread.
+    //
+
+    // If we have been initialized by via InitViaZTN() then all we need to check
+    // is that the config file passed here is the same one passed via the ZTN.
+    // If it isn't, issue a nasty message and return a nil pointer.
+    //
+    if (accSciTokens)  // Already initialzed?
+    {
+        // Verify sameness of config file
+        return (accSciTokens->GetConfigFile() == cfn ? accSciTokens : nullptr);
```

Comment:
You likely want to print an error message as to why you failed so as not to leave the site admin scratching their head..